bucket/dia.json

```diff
@@ -0,0 +1,23 @@
+{
+    "homepage": "http://dia-installer.de",
+    "license": "GPL-2.0-or-later",
+    "version": "0.97.2",
+    "description": "Drawing software",
+    "url": "https://downloads.sourceforge.net/project/dia-installer/dia/0.97.2/dia_0.97.2_win32.zip",
+    "hash": "md5:b985954043b2c0a4664a914de24e9dc7",
+    "extract_dir": "dia",
```

Comment:
```suggestion
```

With this you will just extract lots of dll files

![A](https://i.imgur.com/t9QtHKt.png)